test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1939,6 +1939,19 @@ public void scans() throws Exception {
     ts.exec("deletetable -f t");
   }
 
+  @Test
+  public void scansWithColon() throws Exception {
+    ts.exec("createtable twithcolontest");
+    make10WithColon();
+    ts.exec("scan -r row0 -c c\\:f:col0", true, "value");
+    String result = ts.exec("scan -b row1 -c c\\:f:col1 -e row1");
+    assertEquals(2, result.split("\n").length);
+    result = ts.exec("scan -r row0 -c c:f");
+    assertEquals(1, result.split("\n").length);
```

Comment:
With the latest changes, it seems the use of an '\' character is unnecessary now. The reason your second check doesn't pass is because it does not contain the :col1 at the end.

Reply:
So `scan -b row1 -c c\:f:col1 -e row1` will return the row, `scan -b row1 -c c:f:col1 -e row1` will as well, but `scan -b row1 -c c\:f -e row1` and `scan -b row1 -c c:f -e row1` will not

Reply:
For Some reasons the '\' doesn't make it, the string is not escaped anymore and hence the RE was un-necessary. I made it more simpler. The last test is a negative test I deliberately left out to make sure that things are working. Hope that makes sense.

Reply:
> The last test is a negative test I deliberately left out to make sure that things are working. Hope that makes sense.

I understood that, I was using it to show the inconsistent behavior. My first comment was just poorly worded on my part.